recipes/benchmark-4dn/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "Benchmark-4dn" %}
+{% set version = "0.5.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f44a5d8274710828045955dae92e7ee093174ed88cdc695840f4a250ab9003a3"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: f44a5d8274710828045955dae92e7ee093174ed88cdc695840f4a250ab9003a3
```